pkg/model/crd_util.go

```diff
@@ -0,0 +1,398 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
What is this entirely new file?

Reply:
Just for organization and make navigation easier. `crd.go` for `type CRD` and it's methods; `crd_util.go` for utility functions that interacts with `CRD`s. Also godoc is not generated per go file but per go package - so having multiple files or a single one doesn't impact the generated docs.